pkg/config/env/gcp.go

```diff
@@ -6,12 +6,18 @@ import (
 	"github.com/gobuffalo/envy"
 )
 
-// GcpBucketName returns Google Cloud Storage bucket name defined by ATHENS_STORAGE_GCP_BUCKET
-func GcpBucketName() (string, error) {
+// GCPBucketName returns Google Cloud Storage bucket name defined by ATHENS_STORAGE_GCP_BUCKET
+func GCPBucketName() (string, error) {
 	env, err := envy.MustGet("ATHENS_STORAGE_GCP_BUCKET")
 	if err != nil {
 		return "", fmt.Errorf("missing Google Cloud storage bucket name: %s", err)
 	}
 
 	return env, nil
 }
+
+// GCPPRojectID returns the project id on which the project
+// is running or the cloud storage is using.
+func GCPPRojectID() string {
+	return envy.Get("GCP_PROJECT_ID", "")
```

Comment:
This variable is usually named `GOOGLE_CLOUD_PROJECT`. Some libraries will look for that variable automatically.